stock_disable_force_availability_button/README.rst

```diff
@@ -0,0 +1,16 @@
+Disable force availability button
```

Comment:
Please update this to the latest README template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst